config/routes.rb

```diff
@@ -30,9 +31,13 @@
   }
 
   # if Rails.env.development?
-  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
   # end
 
+  authenticate :user do
+    mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
+    match "/background" => BetterDelayedJobWeb, anchor: false, via: [:get, :post]
```

Comment:
es wird also auf jedem main-server ein ...server.../background geben, richtig? aber nur für eingeloggte admins? wer hat zugriff?
außerdem sorgt das wrappen hier dafür, dass die /graphiql auch nur noch per admin-login erreichbar ist?

die infos hätte ich gerne am commit gehabt in der message

Reply:
korrekt.